CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # How to Contribute
 
-We'd love to accept your patches and contributions to this project. There are
+We'd love to accept your patches ,bug and issue fixes and contributions to this project. There are
```

Comment:
This change has nothing to do with the issue you are addressing. (And also it is typographically incorrect.)